ui/apps/platform/src/hooks/useFeatureFlags.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { createContext, useContext, ReactNode, useMemo } from 'react';
```

Comment:
Off the subject of the change, it will help if `ReactNode` can have `import type` statement.

So we do not need to circle back after changes in #16111